app/app/settings.py

```diff
@@ -577,9 +577,6 @@ def callback(request):
 GITHUB_API_TOKEN = env('GITHUB_API_TOKEN', default='')  # TODO
 GITHUB_APP_NAME = env('GITHUB_APP_NAME', default='gitcoin-local')
 
-# Etherscan API
-ETHERSCAN_API_KEY = env('ETHERSCAN_API_KEY', default='')
```

Comment:
Shouldn't this be kept ?

Reply:
@thelostone-mc That was the original line, however, PR #6940 added the duplicate to line 56, so I removed the original from L581 and kept then one from the PR. Same with the local.env file.

See: https://github.com/gitcoinco/web/blob/82b91f975ffb17f58f94f38dada489de06dd362f/app/app/settings.py#L56